docs/src/main/asciidoc/overview.adoc

```diff
@@ -786,3 +786,20 @@ For example, if you want to gain access to a bean that is defined at the applica
 When the binder discovers that these customizers are available as beans, it will invoke the `configure` method right before creating the consumer and producer factories.
 
 Both of these interfaces also provide access to both the binding and destination names so that they can be accessed while customizing producer and consumer properties.
+
+[[admin-client-config-customization]]
+=== Customizing AdminClient Configuration
+
+As with consumer and producer config customization above, applications can also customize the configuration for admin client by providing an AdminClientConfigCustomizer.
```

Comment:
```suggestion
As with consumer and producer config customization above, applications can also customize the configuration for admin clients by providing an `AdminClientConfigCustomizer`.
```

Reply:
Good points about the bindings/binder names. I will add separate issues for those.

Reply:
I meant all the customizers (producer, consumer) not just the admin.